src/server/models/index.js

```diff
@@ -21,7 +21,8 @@ module.exports = {
     timestamps: false
   }),
   Repo: sequelize.define('repo', {
-    id: { type: Sequelize.INTEGER, primaryKey: true },
+
+    id: { type: Sequelize.STRING(200), primaryKey: true },
```

Comment:
Hey @samueldaviddelacruz! Thanks for contributing to this project, that's awesome! Can you remove the extra empty lines added here. We just want to be consistent with the code style.